.github/workflows/python-app.yml

```diff
@@ -10,25 +10,38 @@ on:
     branches: [ master ]
 
 jobs:
+  
   build:
-
-    runs-on: ubuntu-latest
-
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        version: ["3.8", "3.9", "3.10", "3.11", "3.12", "3.13"]
```

Comment:
I guess it would be enough if we test for the lower and upper bound?

Reply:
genius!